plenum/test/consensus/checkpoint_service/test_update_watermarks_api.py

```diff
@@ -19,6 +21,7 @@ def test_propagate_primary_is_Master_watermarks_not_changed_if_last_ordered_not_
     assert checkpoint_service._data.low_watermark == 0
 
 
+@pytest.mark.skip(reason="INDY-1336. For not watermarks are not dropped before new view")
```

Comment:
Instead of removing, I think it's better to rename the test and change the logic a bit to check that watermarks are NOT dropped now. 
We may need to call methods related to view change finish (or primaryName setter?)

Reply:
i removed this test, because for now we don't have reset_watermarks_before_new_view method